frontend/src/pages/Interventions/data/reducer.ts

```diff
@@ -0,0 +1,29 @@
+import {Action, ActionTypes, InterventionState} from "../../../models/types";
+import {RootStateOrAny} from "react-redux";
+
+const initialState: InterventionState = {
+  currentWordIdx: 0,
+  currentActivityIdx: 0,
+  interventions: null,
+};
+
+const interventionReducer = (state: InterventionState = initialState, action: Action): InterventionState => {
+  const payload = action.payload;
+
+  switch (action.type) {
+    case ActionTypes.GET_WORDS_SUCCESS:
```

Comment:
Is this for `GET_WORDS_SUCCESS`? I think it should be `GET_INTERVENTIONS_SUCCESS`